src/viewer.js

```diff
@@ -2132,6 +2132,58 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
      */
     _cancelPendingImages: function() {
         this._loadQueue = [];
+    },
+
+    /**
+     * Remove the reference strip.
+     * @function
+     * @return {OpenSeadragon.Viewer} Chainable.
+     */
+    removeReferenceStrip: function() {
+        this.showReferenceStrip = false;
+
+        if (this.sequenceMode) {
+            if (this.referenceStrip) {
+                this.referenceStrip.destroy();
+                this.referenceStrip = null;
+            }
+        }
+
+        return this;
+    },
+
+    /**
+     * Display the reference strip based on the currently set tileSources.
+     * @function
+     * @return {OpenSeadragon.Viewer} Chainable.
+     */
+    addReferenceStrip: function() {
+        this.showReferenceStrip = true;
+
+        if (this.sequenceMode) {
```

Comment:
This check is good... we don't want the reference strip if we're not in sequence mode. This restriction should probably be added to the documentation comment above as well. 

It might also be good to add a `$.console.warn` if sequenceMode is false as a warning to the dev that they're calling this function incorrectly.